### PR TITLE
PMM-6645 Issue with vertamedia-clickhouse-datasource plugin

### PR DESCRIPTION
**Pull request by adivinho**
- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/1131

### CODE REVIEW

File: import-dashboards.py [hunk 1]
```diff
@@ -276,7 +276,11 @@ def add_panels():
         for file in files_list:
             with zipfile.ZipFile(os.path.join(GRAFANA_PLUGINS_DIR, file), 'r') as zip_ref:
                 print '    * Unzip %r' % (file,)
-                zip_ref.extractall(GRAFANA_PLUGINS_DIR)
+                for info in zip_ref.infolist():
+                    extracted_path = zip_ref.extract(info, GRAFANA_PLUGINS_DIR)
+                    unix_attributes = info.external_attr >> 16
```
Comment (askomorokhov) [inline import-dashboards.py:281]: Please confirm: this is the right direction of shift? What permissions we should expect?

`-rw-r–r–` == `644` == `0110100100`
```
>>> 0b0110100100 >> 16
0
```

**Review by askomorokhov — CHANGES_REQUESTED** [review]
Looks good to me, but need one confirmation.

File: import-dashboards.py (hunk 1 above)
Comment (adivinho) [reply import-dashboards.py:281]: Confirmed. We are removing extra zeros at the end that are generated by ZipInfo library. 
I have just tested the script with a zip archive of files with different permissions.

Screenshot_20200928_140118

Screenshot_20200928_140159

**Review by askomorokhov — APPROVED** [review]
👍

**Review by AlekSi — APPROVED** [review]
LGTM, but please add a comment explaining why we do that, and a link to the ticket

**Pull request merged**